lib/service/service.go

```diff
@@ -6834,18 +6834,6 @@ func readOrGenerateHostID(ctx context.Context, cfg *servicecfg.Config, kubeBacke
 		if err := persistHostIDToStorages(ctx, cfg, kubeBackend); err != nil {
 			return trace.Wrap(err)
 		}
-	} else if kubeBackend != nil && hostid.ExistsLocally(cfg.DataDir) {
```

Comment:
Looks like we can remove `ExistsLocally` completely as this is the only place it is used.

Reply:
Done